### ISSUE TITLE
A decent lobby

### ISSUE THREAD

**Issue by joecollyer** [SerialFlame/Server#2]
**Describe the solution you'd like**
It would be nice if the build team completed the lobby.

**Additional context**
No

**Servers**
Lobby

**Comment by UntrustWasTaken** [comment]
i agree the current one is a dump especially that sand house  that isnt even a liesure centere
